am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/GetRolePermissionsIntegrationTest.java

```diff
@@ -77,7 +79,7 @@ void setUp() {
     @Test
     void returnListOfPermissionsForRoleName() {
         Map<JsonPointer, Set<Permission>> accessRecord =
-            service.getRolePermissions(buildResource(serviceName, RESOURCE_TYPE, RESOURCE_NAME), ROLE_NAMES);
+            service.getRolePermissions(buildResource(serviceName, resourceType, RESOURCE_NAME), ROLE_NAMES);
```

Comment:
why is buildResource needed here. Can you not pass in resourceDefinition from the setUp?

Reply:
👍

Reply:
👍